build_rln.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# This script is used to build the rln library for the current platform, or download it from the
+# release page if it is available.
+
+set -e
+
+# first argument is the build directory
+build_dir=$1
+
+if [ -z "$build_dir" ]; then
+    echo "No build directory specified"
+    exit 1
+fi
+
+# Get the host triplet
+host_triplet=$(rustup show | grep "Default host: " | cut -d' ' -f3)
+
+# Download the prebuilt rln library if it is available
+if wget https://github.com/vacp2p/zerokit/releases/download/nightly/$host_triplet-rln.tar.gz
```

Comment:
If we are saving such files at repo root we should also also add something like `*.tar.gz` to `.gitignore`.

Reply:
Addressed in [2ea1ca0](https://github.com/waku-org/nwaku/pull/1603/commits/2ea1ca007a0923c8cd07f85ada244a1ee07cf1f6)